client/src/lib.rs

```diff
@@ -4,14 +4,12 @@ use core::fmt;
 use frame_support::{debug, decl_error, decl_event, decl_module, decl_storage, dispatch::DispatchResult, storage::{StorageDecodeLength, StorageMap}, weights::Weight, Hashable};
 use parity_scale_codec::{Encode, Decode};
 
-use frame_system::{Origin, ensure_none, ensure_signed, offchain::{
+use frame_system::{ensure_none, ensure_signed, offchain::{
 		AppCrypto, CreateSignedTransaction,
-		SignedPayload, Signer, SigningTypes, SubmitTransaction,
+		SignedPayload, SigningTypes, SubmitTransaction,
 	}};
 use sp_core::{crypto::KeyTypeId};
-use sp_core::offchain::OpaqueMultiaddr;
 use sp_io::offchain_index;
-use sp_api;
 use sp_runtime::{generic,
```

Comment:
General readability improvement suggestions:-

- Club all non-public `const` together
- Documentation comments (`\\\`) on all enum variants
- Remove commented out code
- Run `cargo fmt` to format the code

Reply:
can I get clarity on the first point, do you mean I should put all the non-public `const` code together?

Reply:
Yes, basically all private `const` together without spaces between them. Something like:-

```rust
/// The keys can be inserted manually via RPC (see `author_insertKey`).
pub const KEY_TYPE: KeyTypeId = KeyTypeId(*b"demo");

// The type to sign and send transactions.
const UNSIGNED_TXS_PRIORITY: u64 = 100;
// TODO: api request
const HTTP_REMOTE_REQUEST: &str = "https://api.pro.coinbase.com/products/ETH-USD/ticker";
const HTTP_HEADER_USER_AGENT: &str = "jaminu71@gmail.com";
const HTTP_ETHEREUM_HOST: &str = "http://127.0.0.1:8080";
const FETCH_TIMEOUT_PERIOD: u64 = 3000; // in milli-seconds
const LOCK_TIMEOUT_EXPIRATION: u64 = FETCH_TIMEOUT_PERIOD + 1000; // in milli-seconds
const LOCK_BLOCK_EXPIRATION: u32 = 3; // in block number
const ONCHAIN_TX_KEY: &[u8] = b"ocw-demo::storage::tx";
```